tests/HttpExtractorTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\HttpExtractor;
```

Comment:
Keboola\HttpExtractor\Tests ?

Reply:
Má to nějakou zásadní výhodu? Takhle to má tu výhodu, že funguje generování testu v Phpstormu bez toho, abych musel něco psát. Jinak musím vždycky doplnit na správné místo v namespace "Tests".

Reply:
ne, asi nema :)

Reply:
mozna ma - myslim, ze pak nefunguje autoload v testech? jakoze pokud je tam treba samostatna mock trida

Reply:
Nn, funguje. Měli jsme to takhle v Drivetu. A tam byly v testech třeba fixtures. Ale pro jistotu jsem to ještě teď vyzkoušel. A funguje. V `composer.json` to mám vyřešené takhle:
```
    "autoload": {
        "psr-4": {
            "Keboola\\HttpExtractor\\": "src/"
        }
    },
    "autoload-dev": {
        "psr-4": {
            "Keboola\\HttpExtractor\\": "tests/"
        }
    },
```

Reply:
tak to mi uslo, ze muze byt autload na jednom namespace ve dvou adresarich (prijde mi to teda divny), tak pak to fungovat bude, tak super :)

Reply:
aha, koukam ze tady je to divne no https://github.com/keboola/docker-demo-app/blob/a9a59f07d4ab131efc76b62b22cd6722ad1650af/composer.json#L13
Delam to spis teda tak ze to ma taky svuj namespace Tests, jako to maji napr. Symfony komponenty https://github.com/keboola/db-extractor-common/blob/master/composer.json#L31
Ale asi to neni problemi mit i takhle.

Reply:
Klidně se podřídím a přidám namespace. Není potřeba to nutně dělat jinak, když to jinde je takhle. Konzistence především.